bitwarden_license/bitwarden-sm/src/projects/create.rs

```diff
@@ -40,7 +40,7 @@ pub(crate) async fn create_project(
     });
 
     let config = client.internal.get_api_configurations().await;
-    let res = bitwarden_api_api::apis::projects_api::organizations_organization_id_projects_post(
+    let res = bitwarden_api_api::apis::projects_api::projects_create(
```

Comment:
Ah sensible function names, how much i've missed you

Reply:
Not sure what you could possibly mean, `organizations_organization_id_projects_post` is soo descriptive 😈